### ISSUE TITLE
Shooting angle for players at low levels

### ISSUE THREAD

**Issue by Regulator13** [famegames/fire-and-ice#35]
**Describe the bug**
When beginning a room, the shooting angle for the character's fire and ice ball is limited. They can either shoot straight up or at approimately a 30 degree angle from the x axis.

**To Reproduce**
1. Play the latest version
2. Single player, any map
3. Without moving, try moving your mouse around to different angles and shooting
4. Most of the bullets will shoot up infinitely, while others will weakly shoot sideways

**Expected behavior**
Fire and Iceballs travel toward the mouse regardless of y position and travel approximately the same length

**Additional context**
Windows 10

**Comment by afrey15** [comment]
This is one of the most infuriating things for me haha. High priority for sure. Also, I don't know if the distance of the cursor from the player affects the strength of the throw, but if so, PLEASE REMOVE. If we want a throw strength variable, we should consider the time between key press and key release being the determining factor for this.

**Comment by Regulator13** [comment]
Interesting, so you want uniform strength for all throws? I'm trying to think of a situation where I've wanted to do a little throw. I guess mostly whenever the direction was going wrong so maybe fixing the direction will remove the need for little throws. Only one way to find out!

**Comment by afrey15** [comment]
It looks like it is uniform strength right now so nothing to change there. I don't think we have a need for little throws at the moment, but it could appear as new elements are added.

**Comment by Regulator13** [comment]
I don't think it's uniform strength right now. If you aim directly above your character the bullets fly off the screen, but sideways they barely move

**Comment by afrey15** [comment]
True, but nothing causes variation outside of the angle itself. We're on the same page still. It needs to be corrected so that they travel the same distance no matter the angle you fire at.

**Comment by Regulator13** [comment]
Oh you're gonna like what I've done with this
